addons/website/static/src/scss/website.editor.ui.scss

```diff
@@ -1,812 +1,803 @@
-// TOP BAR (EDIT)
-#web_editor-top-edit {
-    height: $o-navbar-height;
-    background-color: $o-we-color-dark;
-    z-index: $zindex-modal - 9;
-    position: fixed;
-    top: 0px;
-    right: 0px;
-    width: auto;
-    white-space: nowrap;
-
-    background-color: rgba(0, 0, 0, 0);
-    transition: background-color 400ms $o-we-md-ease 0s;
-    font-family: $o-we-font-family;
-
-    form.navbar-form {
-        height: $o-navbar-height;
-        z-index: 1060;
-        margin: 0;
-        padding: 0;
-        @include o-position-absolute($top: -1px, $right: -$o-we-sidebar-width);
-        transition: right 0.4s $o-we-md-ease 0s;
-        border-left: 1px solid $o-we-color-divider;
-        background-color: inherit;
+body.editor_enable {
+    overflow: hidden;
+    padding-top: 0 !important;
 
-        .btn-group {
-            height: 100%;
-        }
+    // TOP BAR (EDIT)
+    #web_editor-top-edit {
+        height: $o-navbar-height;
```

Comment:
It is necessary, because there is forum. Otherwise, in the case of the forum page that has a field html, the publisher does not have the correct color.